content/e-services/1-build.md

```diff
@@ -6,12 +6,15 @@ weight: 1
 
 ### Build
 
-Core at heart we are builders, who practice craft of making software. We use computing technologies to solve problems for humans.
+Core at heart we are builders.
```

Comment:
Doesn't `core` imply `heart`?

Reply:
:-) Not sure. Let me lookup what Internet says.